docs/Formula-Cookbook.md

```diff
@@ -175,7 +175,7 @@ description can be overridden using the normal option syntax (in this case, the
     ```
 * Some [`Requirement`](https://rubydoc.brew.sh/Requirement)s can also take a string specifying their minimum version that the formula depends on.
 
-**Note:** [`option`](https://rubydoc.brew.sh/Formula#option-class_method)s are not allowed in Homebrew/homebrew-core as they are not tested by CI.
+**Note:** [`:optional`](https://rubydoc.brew.sh/Formula#option-class_method)s are not allowed in Homebrew/homebrew-core as they are not tested by CI.
```

Comment:
This isn't quite the same. `option` is a thing that we no longer support. `:recommended` and `:optional` aren't supported and they should be mentioned, I agree, but `option` needs to stay.

Reply:
There is another note about `option`s further down the document, is that what you are thinking about?

While the link in this change does point to a section about `option` it also mentions `:optionals` so I thought it was good enough.

Apologies if I don't understand you concern.

Reply:
> While the link in this change does point to a section about `option` the section also mentions `:optionals` so I thought it was good enough.

Ok. In that case:
- it should be changed to reference both `:optional` and `:recommended`
- the rubydoc.brew.sh link should be correctly updated or removed (it's `:optional` pointing to `option` currently)

Reply:
Got it - thanks :-)